xbmc/video/VideoDatabase.h

```diff
@@ -945,11 +945,7 @@ class CVideoDatabase : public CDatabase
   void SetArtForItem(int mediaId, const MediaType &mediaType, const std::string &artType, const std::string &url);
   void SetArtForItem(int mediaId, const MediaType &mediaType, const std::map<std::string, std::string> &art);
   bool GetArtForItem(int mediaId, const MediaType &mediaType, std::map<std::string, std::string> &art);
-  std::string GetArtForItem(int mediaId, const MediaType &mediaType, const std::string &artType);
-  bool GetArtForAsset(int assetId,
-                      int ownerId,
-                      const MediaType& mediaType,
-                      std::map<std::string, std::string>& art);
+  std::string GetArtForItem(int mediaId, const MediaType& mediaType, const std::string& artType);
```

Comment:
Could this method be `const`? I don't think it changes state?

Reply:
Oddly it could, but through a bit of unvoluntary trickery I believe. m_pDS2 is modified in the end.
I didn't touch that function because I haven't seen it called for movies and only "TV recently added" but it would be safer to change it now as well.

Reply:
Actually there also are RemoveArtForItem() function to take care of, as well as HasArtForItem()